network-api/networkapi/wagtailpages/pagemodels/customblocks/profile_directory.py

```diff
@@ -45,3 +56,135 @@ def get_context(self, value, parent_context=None):
     class Meta:
         template = 'wagtailpages/blocks/profile_directory.html'
         icon = 'group'
+
+
+class TabbedProfileDirectory(blocks.StructBlock):
+    """
+    This component is set up to create a directory with tabs. The tabs are filtered on a
+    specific profile field (e.g. profile_type).
+    https://github.com/mozilla/foundation.mozilla.org/issues/7422
+    """
+    tabs = SnippetChooserBlock(
+        'wagtailpages.PulseFilter',
+        help_text=(
+            'Tabs are created based on the selected pulse filter and the first option in the snippet will be the first tab showing open on the page.'
```

Comment:
@richbrennan This line seemed to be over the character limit for the back-end tests.
```suggestion
            'Tabs are created based on the selected pulse filter ',
            'and the first option in the snippet will be the first tab showing open on the page.'
```

 Once I updated this on my local machine, I then got a error about a mozfest/0031_... migration missing. But since I saw a 0031 migration already exists, I figured you might know what the test is referencing. Thanks in advance!

Reply:
I updated this help text for now but if there is some sort of migration conflict, that might be better for @richbrennan to handle. 😇

Reply:
Hey @stevedya 

I suspect `main` branch merging weirdness, now fixed.